ui/app/models/job.js

```diff
@@ -37,13 +37,7 @@ export default class Job extends Model {
 
   @computed('plainId')
   get idWithNamespace() {
-    const namespaceId = this.belongsTo('namespace').id();
-
-    if (!namespaceId || namespaceId === 'default') {
-      return this.plainId;
-    } else {
-      return `${this.plainId}@${namespaceId}`;
-    }
+    return `${this.plainId}@${this.belongsTo('namespace').id()}`;
```

Comment:
Relying on `this.belongsTo('namespace').id()` to have a value scares me a little bit. I think it always should but it might be nice to still be defensive here and slip a `?? 'default'` in.

Reply:
Good call